@maskedmaxx Looks like there is a new linting error: "toxics/packet_loss.go:33:1: The line is 128 characters long, which exceeds the maximum of 100 characters."

--- a/toxics/packet_loss.go
+++ b/toxics/packet_loss.go
@@ -30,7 +30,7 @@ type PacketLossToxic struct {
 // per new connection so every connection gets its own RNG and drop state.
 func (t *PacketLossToxic) NewState() interface{} {
 	return &PacketLossToxicState{
-		rng: rand.New(rand.NewSource(rand.Int63())), // #nosec G404 -- packet loss simulation does not need cryptographic randomness
+		rng: rand.New(rand.NewSource(rand.Int63())), // #nosec G404 -- packet loss simulation
 	}
 }
 

